### PR TITLE
Diff left

### PR DESCRIPTION
**Pull request by atrovato**
Pending files from #121

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+npmpublish:
```
Comment (rzr) [inline .gitlab-ci.yml:1]: do you plan to relocate to gitlab ?

Comment (atrovato) [reply .gitlab-ci.yml:1]: Thanks for reading!
This PR contains only diff left after splitting the big #121
So I'll remove this ;)

**Comment by rzr** [comment]
should this be merged for next release ?

**Comment by atrovato** [comment]
No, I need to check if this is still relevant first.

**Comment by rzr** [comment]
can this change be part of next release ?

**Comment by atrovato** [comment]
Nop, this is still draft (PR is marked as draft).

**Comment by rzr** [comment]
no problem I was just thinking making a release soon I can wait for you couple of weeks more

**Comment by rzr** [comment]
Is anyone able to resume those stalling works ?

**Pull request closed without merging**